pkg/component/gardener/discoveryserver/discoveryserver_test.go

```diff
@@ -720,6 +691,7 @@ qwIDAQAB
 			})
 
 			It("should successfully deploy all resources", func() {
+				format.MaxLength = 0
```

Comment:
Is this a left-over from development?

Reply:
Yes, thanks for catching it.

Reply:
Fixed with [0014664](https://github.com/gardener/gardener/pull/10538/commits/001466454ea0ac23b24fcc842ee9ae76ae3c96fb).